src/test/java/ladder/domain/ladder/LadderTest.java

```diff
@@ -0,0 +1,15 @@
+package ladder.domain.ladder;
+
+import ladder.domain.util.RightPointRandom;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+class LadderTest {
+
+    @Test
+    void getSize() {
+        Ladder ladder = new Ladder(5, 3, new RightPointRandom());
+        Assertions.assertThat(ladder.getLines().size()).isEqualTo(3);
```

Comment:
`assertj `검증문을 활용해도 좋을 것 같아요.

```suggestion
        assertThat(ladder.getLines()).hasSize(3);
```